javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,39 @@
+//Student manager
+//Use Array.push
+
+const class07Students = [];
+function addStudentToClass (studentName) {
+  if (!studentName) return;                            //If no student name is given then return 
```

Comment:
What if `studentName = "    "` empty string.
- try to check if it is an empty string (use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim))